app/utils/background.py

```diff
@@ -120,42 +153,23 @@ def fetch_new_awards_from_date(
     while contracts_response.json():
         total += len(contracts_response_json)
         for entry in contracts_response_json:
-            with contextmanager(db_provider)() as session:
-                with transaction_session_logger(session, "Error creating the application"):
-                    award = _create_award(entry, session)
-                    borrower = _get_or_create_borrower(entry, session)
-                    award.borrower_id = borrower.id
-
-                    application = _create_application(
-                        award.id,
-                        borrower.id,
-                        borrower.email,
-                        borrower.legal_identifier,
-                        award.source_contract_id,
-                        session,
-                    )
-
-                    message = models.Message.create(
-                        session,
-                        application=application,
-                        type=models.MessageType.BORROWER_INVITATION,
-                    )
-
-                    message_id = mail.send_invitation_email(
-                        sesClient,
-                        application.uuid,
-                        borrower.email,
-                        borrower.legal_name,
-                        award.buyer_name,
-                        award.title,
-                    )
-                    message.external_message_id = message_id
+            _create_complete_application(entry, db_provider)
         index += 1
         contracts_response = data_access.get_new_contracts(index, last_updated_award_date, until_date)
         contracts_response_json = contracts_response.json()
     logger.info("Total fetched contracts: %d", total)
 
 
+def fetch_award_by_contract_and_supplier(
+    contract_id: str, supplier_id: str, db_provider: Callable[[], Generator[Session, None, None]]
+) -> None:
+    contract_response = data_access.get_contract_by_contract_and_supplier(contract_id, supplier_id).json()
+    if not contract_response:
+        logger.info(f"The contract with id {contract_id} and supplier id {supplier_id} was not found")
```

Comment:
Typically info level will not print, when called from CLI. Maybe have this function return a boolean, and then print a success or error message in commands.py?

Reply:
We could open an issue to change this for all the commands as they all are this way. But for this particular case, this command is meant to be run manually only, so I don't think it matters.